packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,659 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Netlify TOML config schema",
+  "description": "Config file for Netlify",
+  "type": "object",
+  "definitions": {
+    "BasePath": {
+      "title": "Base path",
+      "description": "Directory to change to before starting a build. This is where we will look dependency management files such as `package.json` or `.nvmrc`. If not set, defaults to the root directory.",
```

Ah, I must have made a typo in my suggestion, sorry. We're missing a word here

```suggestion
      "description": "Directory to change to before starting a build. This is where we will look for dependency management files such as `package.json` or `.nvmrc`. If not set, defaults to the root directory.",
```